rootfs/docker-entrypoint-initdb.d/001_setup_envdir.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+cd "$WALE_ENVDIR"
+
+# access-key-id and access-secret-key files are mounted in via kubernetes secrets
+AWS_ACCESS_KEY_ID=$(cat access-key-id)
+AWS_SECRET_ACCESS_KEY=$(cat access-secret-key)
+
+# setup envvars for wal-e
+cp access-key-id AWS_ACCESS_KEY_ID
+cp access-secret-key AWS_SECRET_ACCESS_KEY
```

Comment:
Are you copying the fancy boto name to a different format? Instead of having it mounted twice via k8s?